In [3]:
p = 1/6
p
1 - 125/216

0.4212962962962963

# Begriffe 
<dl>
 <dt>Relative Häufigkeit (Bezug: Data Science)</dt>
 <dd>En: <var>Relative frequency</var>

Anteil eines Merkmals an der Gesamtzahl (%-uales Vorkommen)
<blockquote cite="https://de.wikipedia">
„Die relative Häufigkeit ist eine Gliederungszahl und ein Maß der deskriptiven Statistik.“
<p>-- WP</p>  
</blockquote>
</dd>

<dt>Zufallsvariable </dt>
 <dd>

 -> besser: Zufallsgröße

</dd>

<dt>Wahrscheinlichkeit</dt>
 <dd> 
 {1,2,3,4,5,6}
 6 Würfel 
 
 Wahrscheinlichkeit: $\dfrac{1}{6} \longrightarrow$ 
 - $\dfrac{1}{Grundgesamtheit}$
 

 Drei Würfe, um eine Sechs zu würfeln:

 $ p = \dfrac{1}{6} + \dfrac{5}{6} \times \dfrac{1}{6} + \dfrac{5}{6} \times \dfrac{5}{6} \times \dfrac{1}{6}$

 $ p = \dfrac{1}{6} + \dfrac{5}{36} + \dfrac{25}{216} $

 $ p \approx 42\%$

„Gegenwahrscheinlichkeit“

 $ p = \dfrac{5}{6} \times \dfrac{5}{6} \times \dfrac{5}{6} $
 $ p = 0.5787037037037037 \approx 58 \%$
 
$  58 \% + 42\% = 1 $

 
 </dd>
<dt>Wahrscheinlichkeitsdichte</dt>
 <dd>
    - Foo
 </dd>

<dt>Erwartungswert / Mittelwert</dt>
 <dd>  
 - BAR:
 
 $\mu =  \dfrac{1+2+3+4+5+6}{6}$
 </dd>

<dt>Varianz</dt>
 <dd> - baz</dd>

 <dt>Standardabweichung</dt>
 <dd> - murX</dd>




</dl>